crypto_sign/rainbowIIIc-classic/clean/blas_comm.c

```diff
@@ -0,0 +1,153 @@
+/// @file blas_comm.c
+/// @brief The standard implementations for blas_comm.h
+///
+
+#include "blas.h"
+#include "blas_comm.h"
+#include "gf.h"
+#include "rainbow_config.h"
+
+#include <stdint.h>
+#include <string.h>
+
+
+void PQCLEAN_RAINBOWIIICCLASSIC_CLEAN_gf256v_set_zero(uint8_t *b, unsigned _num_byte) {
+    gf256v_add(b, b, _num_byte);
+}
+/// @brief get an element from GF(256) vector .
+///
+/// @param[in]  a         - the input vector a.
+/// @param[in]  i         - the index in the vector a.
+/// @return  the value of the element.
+///
+uint8_t PQCLEAN_RAINBOWIIICCLASSIC_CLEAN_gf256v_get_ele(const uint8_t *a, unsigned i) {
+    return a[i];
+}
+
+unsigned PQCLEAN_RAINBOWIIICCLASSIC_CLEAN_gf256v_is_zero(const uint8_t *a, unsigned _num_byte) {
+    uint8_t r = 0;
+    while ( _num_byte-- ) {
```

Comment:
there are extra spaces:

```suggestion
    while (_num_byte--) {
```